x/auth/vesting/types/vesting_account.go

```diff
@@ -1110,6 +1110,159 @@ func (va *ClawbackVestingAccount) clawback(ctx sdk.Context, dest sdk.AccAddress,
 	return nil
 }
 
+// forceTransfer sends up to amt  assets to the dest account.
+// It will first send spendable coins, then unbonding tokens,
+// and finally staked tokens. The tokens must not be otherwise
+// encumbered - e.g. they cannot be locked or unvested.
+// Any changes to the account encumbrance must be written to the store
+// before calling this method so that bank.SendCoins() will work.
+// Any unbonding or staked tokens transferred are deducted from
+// the DelegatedFree/Vesting fields.
+// TODO: refactor to de-dup with clawback()
+func (va *BaseVestingAccount) forceTransfer(ctx sdk.Context, amt sdk.Coins, dest sdk.AccAddress, ak AccountKeeper, bk BankKeeper, sk StakingKeeper) error {
+	if amt.IsZero() {
+		return nil
+	}
+	addr := va.GetAddress()
+
+	// Now that future vesting events (and associated lockup) are removed,
+	// the balance of the account is unlocked and can be freely transferred.
+	spendable := bk.SpendableCoins(ctx, addr)
+	toXfer := coinsMin(amt, spendable)
+	err := bk.SendCoins(ctx, addr, dest, toXfer)
+	if err != nil {
+		return err // shouldn't happen, given spendable check
```

Comment:
This should panic, for clarity that the vesting account state won't be changed on failure.
```suggestion
		panic(err) // shouldn't happen, given spendable check
```

Reply:
Done.